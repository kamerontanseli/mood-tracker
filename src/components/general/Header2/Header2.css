.Header2 {
  font-weight: 300;
  color: #333;
}