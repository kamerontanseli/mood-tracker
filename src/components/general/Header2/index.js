import "./Header2.css";
import React from "react";

const Header2 = props => <h2 className="Header2" {...props} />

export default Header2;
