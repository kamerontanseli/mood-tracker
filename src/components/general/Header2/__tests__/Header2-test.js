import React from 'react'
import renderer from 'react-test-renderer'
import Header2 from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <Header2>Hello world</Header2>
    );
    expect(snapshot).toMatchSnapshot()
  })
})