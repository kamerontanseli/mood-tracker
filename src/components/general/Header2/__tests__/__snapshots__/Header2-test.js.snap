// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<h2
  className="Header2"
>
  Hello world
</h2>
`;
